apps/web/lib/settings/member-table.tsx

```
@@ -1,268 +1,309 @@
+
+	const handelNameChange = useCallback(
+		(event: ChangeEvent<HTMLInputElement>) => {
+			const name = event.target.value || '';
+			if (name === editMemberRef.current?.employee.fullName) {
+				return;
+			}
+
+			const names = name.split(' ');
+			const tempMember: OT_Member | null = cloneDeep(editMemberRef.current);
+
+			if (tempMember && tempMember.employee.user) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)